demisto_sdk/commands/validate/validators/IN_validators/IN164_is_new_required_param_no_default.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
+from demisto_sdk.commands.content_graph.objects.integration import (
+    Integration,
+    Parameter,
+)
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class IsNewRequiredParamNoDefaultIntegrationValidator(BaseValidator[ContentTypes]):
+    error_code = "IN164"
+    description = "Ensure that new *required* parameters added to an existing integration must have a default value."
+    rationale = (
+        "Adding a new required parameter or changing a non-required one to required without specifying a "
+        "default value breaks backward compatibility and will prevent users from upgrading their "
+        "integration instances."
+    )
+    error_message = (
+        "Possible backward compatibility break: "
+        "You have added the following new *required* parameters: {param_list}. "
+        "Please undo the changes or provide default values. If cannot give default value, "
+        "please make the parameter not required and check in code implementation that it was configured."
+    )
+    related_field = "configuration"
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        results: List[ValidationResult] = []
+
+        for content_item in content_items:
+            old_content_item = content_item.old_base_content_object
+
+            # If there is no old content item, this is a new integration so there is no issue creating a required param
+            if not old_content_item:
+                continue
```

Comment:
we can assume there is old_content_item.

Reply:
how can we always assume that? what if its a new integration?

Reply:
but your validation contains the filed 
`expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]`
which filters out added integrations. Also, added integrations has old_content_item, which is a copy of themself